src/main/kotlin/com/mineinabyss/staminaclimb/component/ClimbingEquipment.kt

```diff
@@ -0,0 +1,8 @@
+package com.mineinabyss.staminaclimb.component
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+@SerialName("climb:equipment")
+data class ClimbingEquipment(val modifier: Double)
```

Comment:
I think it's better to have a more direct name, maybe `"climb:stamina_multiplier"` and an appropriate class and extension name. You can also use a `value class` to avoid having to specify `modifier` in the component configuration